wiki/Modding_Guides/Getting_Your_Map_Modded/zh.md

```diff
@@ -39,7 +42,7 @@ Modding Queues 版块
 
 ### Pending吧
 
-传送门：http://tieba.baidu.com/f?kw=pending
+传送门：https://tieba.baidu.com/f?kw=pending
```

Comment:
【 the browser do not automaticly use http when https is not valid 】
【 and in fact, https://tieba.baidu.com/p/4608591889 works 】